.github/workflows/.spellcheck-conf.toml

```diff
@@ -1,3 +1,3 @@
 [default]
 # Don't correct the "ASSERTne" word
-extend-ignore-words-re = ["ASSER", "Tne"]
+extend-ignore-words-re = ["ASSER", "Tne", "ba", "BA"]
```

Comment:
I guess you can update the comment above 😉

Reply:
Done